### DISCUSSION TITLE
About how to integrate into Vue

### DISCUSSION THREAD

**Discussion by AZCodingAccount** [scalar/scalar#3070] [Q&A] [answered]
How can I pass my JSON string to the ApiReference component? Currently, the URL cannot be directly accessed, and I have manually obtained this JSON string. What configuration option should I pass? I'm using Vue.
code: 
` <ApiReference
      v-if="apiSpec"
      :configuration="{
      spec: apiSpec,
    }"
  />`

**Comment by hanspagel** [comment, marked as answer]
try `{ spec: { content: apiSpec } }` :)

**Comment by AZCodingAccount** [comment]
thx
